codechecker_common/review_status_handler.py

```diff
@@ -49,7 +49,8 @@ class ReviewStatusHandler:
         'checker_filter',
         'report_hash_filter',
         'review_status',
-        'message']
+        'message',
+        'ignore']
```

Comment:
I know I'm nitpicking, but did we not end up deciding that `ignore` is not a review status? So why is it implemented in this file?